lib/screens/organization_list/widgets/organization_card.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:flutter/material.dart';
+
+class OrganizationCard extends StatelessWidget {
```

Comment:
You can make this class more robust
create 2 final fields, String name and position
and accept them using a constructor.
Then you pass these parameters in the screen class